syntax = "proto2";
import "nanopb.proto";
message SimpleMessage {
  required uint32 id = 1;
  required uint32 timestamp = 2;
  required uint32 phase = 3;
  repeated int32 int32value = 4;
  required int32 sample00 = 5;
  required int32 sample01 = 6;
  required int32 sample02 = 7;
  required int32 sample03 = 8;
  required int32 sample04 = 9;
  required int32 sample05 = 10;
  required int32 sample06 = 11;
  required int32 sample07 = 12;
  required int32 sample08 = 13;
  required int32 sample09 = 14;
  required int32 sample10 = 15;
  required int32 sample11 = 16;
  required int32 sample12 = 17;
  required int32 sample13 = 18;
  required int32 sample14 = 19;
  required int32 sample15 = 20;
  required int32 sample16 = 21;
  required int32 sample17 = 22;
  required int32 sample18 = 23;
  required int32 sample19 = 24;
  required int32 sample20 = 25;
  required int32 sample21 = 26;
  required int32 sample22 = 27;
  required int32 sample23 = 28;
  required int32 sample24 = 29;
  required int32 sample25 = 30;
  required int32 sample26 = 31;
  required int32 sample27 = 32;
  required int32 sample28 = 33;
  required int32 sample29 = 34;
  required int32 sample30 = 35;
  required int32 sample31 = 36;
  required int32 sample32 = 37;
  required int32 sample33 = 38;
  required int32 sample34 = 39;
  required int32 sample35 = 40;
  required int32 sample36 = 41;
  required int32 sample37 = 42;
  required int32 sample38 = 43;
  required int32 sample39 = 44;
  required int32 sample40 = 45;
  required int32 sample41 = 46;
  required int32 sample42 = 47;
  required int32 sample43 = 48;
  required int32 sample44 = 49;
  required int32 sample45 = 50;
  required int32 sample46 = 51;
  required int32 sample47 = 52;
  required int32 sample48 = 53;
  required int32 sample49 = 54;
  required int32 sample50 = 55;
  required int32 sample51 = 56;
  required int32 sample52 = 57;
  required int32 sample53 = 58;
  required int32 sample54 = 59;
  required int32 sample55 = 60;
  required int32 sample56 = 61;
  required int32 sample57 = 62;
  required int32 sample58 = 63;
  required int32 sample59 = 64;
};